VORP-Housing/VORP.Housing.Server/Init.cs

```diff
@@ -0,0 +1,315 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VORP.Housing.Server.Extensions;
+using VORP.Housing.Server.Scripts;
+using VORP.Housing.Shared;
+using VORP.Housing.Shared.Models;
+using VORP.Housing.Shared.Models.Json;
+
+namespace VORP.Housing.Server
+{
+    public class Init : Manager
+    {
+        private readonly ConfigurationSingleton _configurationInstance = ConfigurationSingleton.Instance;
+
+        public static Dictionary<uint, House> HousesDb = new Dictionary<uint, House>();
+        public static Dictionary<int, Room> RoomsDb = new Dictionary<int, Room>();
+        
+        public void Initialize()
+        {
+            AddEvent("vorp_housing:BuyHouse", new Action<Player, uint, double>(BuyHouseAsync));
+            AddEvent("vorp_housing:BuyRoom", new Action<Player, int, double>(BuyRoomAsync));
+            AddEvent("vorp_housing:changeDoorState", new Action<uint, bool>(ChangeDoorState));
+            AddEvent("vorp_housing:getRooms", new Action<int>(GetRoomsAsync));
+            AddEvent("vorp_housing:getHouses", new Action<int>(GetHousesAsync));
```

Comment:
Event naming I believe we should discuss if it should either have "EventHandler" in the method name, or have "On" as a prefix for an event handler method. This is for all C# projects, not just this one.

I've been using "On" prefixing, but a discussion is best having.

Reply:
I wouldn't mind using the "On" prefix. I haven't written as many event handlers as I've should.

Reply:
Never mind, based on [the official docs from Microsoft](https://docs.microsoft.com/en-us/dynamicsax-2012/developer/naming-conventions-delegates-and-event-handlers), an "EventHandler" suffix is the suggested way

Reply:
That is the X++ documentation, its fine to use EventHandler, the C# naming conventions isn't clear about it either. The "On" handling is from ASP .NET event naming e.g. `btnSubmit_OnClick`

Reply:
Could be more, we decide what we're happier with; https://stackoverflow.com/questions/724085/events-naming-convention-and-style